### PR TITLE
Translator page clean up

### PR DESCRIPTION
**Pull request by shifucun**
- Remove old Observation table example
- Clean up unused files
- Remove stale BQ link

### CODE REVIEW

File: static/js/translator/page.tsx [hunk 1]
```diff
@@ -140,10 +132,6 @@ export class Page extends React.Component<PagePropType, PageStateType> {
           bindings={this.state.bindings}
           constraints={this.state.constraints}
         ></Translation>
-        <a href="https://console.cloud.google.com/bigquery?sq=443333369001:c5d2972c96704d65a3091ee15d6f0c87">
```
Comment (pradh) [inline static/js/translator/page.tsx:143]: Are we removing BQ link in general?  That seems useful?

File: static/js/translator/page.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ export class Page extends React.Component<PagePropType, PageStateType> {
           <label id="select-input">
             <input
               type="radio"
-              value="place"
+              value="observation"
```
Comment (pradh) [inline static/js/translator/page.tsx:94]: nit: s/Place Observation/Observation/ (below)?

**Review by pradh — APPROVED** [review]
Thanks for the quick fix!

File: static/js/translator/page.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/translator/page.tsx:143]: This link can not update with the actual query which is misleading here. I want to update the translator here to use a real table name so we can copy over the SQL into GCP console.

File: static/js/translator/page.tsx (hunk 2 above)
Comment (shifucun) [reply static/js/translator/page.tsx:94]: Done

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**